src/calendar/index.js

```diff
@@ -75,7 +75,9 @@ class Calendar extends Component {
     // Handler which gets executed when press arrow icon left. It receive a callback can go back month
     onPressArrowLeft: PropTypes.func,
     // Handler which gets executed when press arrow icon left. It receive a callback can go next month
-    onPressArrowRight: PropTypes.func
+    onPressArrowRight: PropTypes.func,
+    // Handler which gets executed when press on Year / Month in header
+    onPressYear: PropTypes.func
```

Comment:
Why this component named 'onPressYear'? The press is on both year and month. How about 'onPressHeader'?